zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -113,6 +145,18 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.ac_frequency_max.name,
             config=REPORT_CONFIG_DEFAULT,
         ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.power_factor.name,
+            config=REPORT_CONFIG_OP,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.power_factor_ph_b.name,
+            config=REPORT_CONFIG_OP,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.power_factor_ph_c.name,
+            config=REPORT_CONFIG_OP,
+        ),
```

Comment:
All power factor attributes do not support attribute reporting according to ZCL spec.